reactor-core/src/main/java/reactor/core/scheduler/Schedulers.java

```diff
@@ -1024,11 +1090,44 @@ default Scheduler newBoundedElastic(int threadCap, int queuedTaskCap, ThreadFact
 		 * @param queuedTaskCap maximum number of tasks to enqueue when no more threads can be created. Can be {@link Integer#MAX_VALUE} for unbounded enqueueing.
 		 * @param threadFactory a {@link ThreadFactory} to use each thread initialization
 		 *
+		 * @since 3.6.0
+		 *
+		 * @deprecated in favor of
+		 * {@link #newBoundedElasticThreadPerTask(int, int, ThreadFactory)}.
+		 * Should be safely removed in 3.8.0
+		 *
 		 * @return a new {@link Scheduler} that dynamically creates workers with an upper bound to
 		 * the number of backing threads
 		 */
 		default Scheduler newThreadPerTaskBoundedElastic(int threadCap, int queuedTaskCap,	ThreadFactory threadFactory) {
```

Comment:
Now we have both threadPerTaskBoundedElastic and boundedElasticPerTask... Even though the latter is more favourable, with the fact that we already released threadPerTaskBoundedElastic, I think it's just better to leave one variant.